modules/gamma/main.nf

```diff
@@ -0,0 +1,40 @@
+def VERSION = '2.1' // Version information not provided by tool on CLI
+
+process GAMMA {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::gamma=2.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gamma%3A2.1--hdfd78af_0':
+        'quay.io/biocontainers/gamma:2.1--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path(db)
+
+    output:
+    tuple val(meta), path("*.gamma"), emit: gamma
+    tuple val(meta), path("*.psl")  , emit: psl
+    tuple val(meta), path("*.gff") , optional:true , emit: gff
```

Comment:
Might be good to check the alignments of `emit` and `optional` as so not to upset certain people ;)